packages/create-snap/README.md

```diff
@@ -0,0 +1,28 @@
+# @metamask/create-snap
+
+A CLI for creating MetaMask Snaps.
+
+## Usage
+
+Use this command line tool directly with `yarn create` or `npx` to create a new snap project:
+
+```sh
+yarn create @metamask/snap your-snap-name
```

Comment:
```suggestion
yarn create @metamask/create-snap your-snap-name
```

Reply:
nop. `yarn create` works differently with namespaced packages. the `create-` part is removed. i like it, it reads much better like that, "Yarn create metamask snap". NPM also works the same way since version 6.